### PR TITLE
build only python

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
Hi @xhochy 
Could you please provide any recommendations how to revolve the issue with:
```
2023-07-11T22:32:36.5616535Z Setting up cross-python
2023-07-11T22:32:36.6348361Z find: ‘/home/conda/feedstock_root/build_artifacts/openvino_1689109588372/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/’: No such file or directory
2023-07-11T22:32:37.6190903Z Traceback (most recent call last):
2023-07-11T22:32:37.6192655Z   File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
2023-07-11T22:32:37.6193138Z     sys.exit(main())
2023-07-11T22:32:37.6193535Z   File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 256, in main
2023-07-11T22:32:37.6201171Z     call_conda_build(action, config)
2023-07-11T22:32:37.6202573Z   File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
2023-07-11T22:32:37.6203268Z     result = api.build(
2023-07-11T22:32:37.6204090Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 253, in build
2023-07-11T22:32:37.6209735Z     return build_tree(
2023-07-11T22:32:37.6210441Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3799, in build_tree
2023-07-11T22:32:37.6238280Z     packages_from_this = build(
2023-07-11T22:32:37.6238891Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2875, in build
2023-07-11T22:32:37.6239186Z     newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
2023-07-11T22:32:37.6239597Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1936, in bundle_conda
2023-07-11T22:32:37.6239834Z     utils.check_call_env(
2023-07-11T22:32:37.6240203Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 450, in check_call_env
2023-07-11T22:32:37.6249240Z     return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
2023-07-11T22:32:37.6250601Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 426, in _func_defaulting_env_to_os_environ
2023-07-11T22:32:37.6250959Z     raise subprocess.CalledProcessError(proc.returncode, _args)
2023-07-11T22:32:37.6251606Z subprocess.CalledProcessError: Command '['/usr/bin/bash', '-e', '/home/conda/feedstock_root/build_artifacts/openvino_1689109588372/work/install/install_dev.sh']' returned non-zero exit status 1.
2023-07-11T22:32:44.7999199Z ##[error]Bash exited with code '1'.
2023-07-11T22:32:44.8028389Z ##[section]Finishing: Run docker build
```
It happens during last output (`openvino` metapackage) build on both cross-compilation targets arm64. 
Looks like standard cross python package cannot initialize itself.

**Comment by ilya-lavrenov** [comment]
> Hi @xhochy Could you please provide any recommendations how to revolve the issue with:
> 
> ```
> 2023-07-11T22:32:36.5616535Z Setting up cross-python
> 2023-07-11T22:32:36.6348361Z find: ‘/home/conda/feedstock_root/build_artifacts/openvino_1689109588372/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/’: No such file or directory
> 2023-07-11T22:32:37.6190903Z Traceback (most recent call last):
> 2023-07-11T22:32:37.6192655Z   File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
> 2023-07-11T22:32:37.6193138Z     sys.exit(main())
> 2023-07-11T22:32:37.6193535Z   File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 256, in main
> 2023-07-11T22:32:37.6201171Z     call_conda_build(action, config)
> 2023-07-11T22:32:37.6202573Z   File "/opt/conda/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
> 2023-07-11T22:32:37.6203268Z     result = api.build(
> 2023-07-11T22:32:37.6204090Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 253, in build
> 2023-07-11T22:32:37.6209735Z     return build_tree(
> 2023-07-11T22:32:37.6210441Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3799, in build_tree
> 2023-07-11T22:32:37.6238280Z     packages_from_this = build(
> 2023-07-11T22:32:37.6238891Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2875, in build
> 2023-07-11T22:32:37.6239186Z     newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
> 2023-07-11T22:32:37.6239597Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1936, in bundle_conda
> 2023-07-11T22:32:37.6239834Z     utils.check_call_env(
> 2023-07-11T22:32:37.6240203Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 450, in check_call_env
> 2023-07-11T22:32:37.6249240Z     return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
> 2023-07-11T22:32:37.6250601Z   File "/opt/conda/lib/python3.10/site-packages/conda_build/utils.py", line 426, in _func_defaulting_env_to_os_environ
> 2023-07-11T22:32:37.6250959Z     raise subprocess.CalledProcessError(proc.returncode, _args)
> 2023-07-11T22:32:37.6251606Z subprocess.CalledProcessError: Command '['/usr/bin/bash', '-e', '/home/conda/feedstock_root/build_artifacts/openvino_1689109588372/work/install/install_dev.sh']' returned non-zero exit status 1.
> 2023-07-11T22:32:44.7999199Z ##[error]Bash exited with code '1'.
> 2023-07-11T22:32:44.8028389Z ##[section]Finishing: Run docker build
> ```
> 
> It happens during last output (`openvino` metapackage) build on both cross-compilation targets arm64. Looks like standard cross python package cannot initialize itself.
> 
> Logs are here https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=743772&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=ced5d8de-8227-5f3f-33a1-45cf1592c45a&t=1f4c3150-ddc9-52b5-8700-e07f78624442

@isuruf could you please advice?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -56,38 +35,22 @@ requirements:
...
     - ninja
+    - ccache
     - pkg-config # to generate openvino.pc on macOS
-    - scons  # [aarch64 or arm64]
     - cmake >=3.15 # to have new cmake cmd interface (-S, --install and other options)
-    - python >=3.7
-    - libprotobuf =3.20.3 # for protoc compiler, libraries are used from libprotobuf-static
-    - flatbuffers =23.3.3
     - sysroot_linux-64 =2.17  # [linux64]
   host:
```
Comment (xhochy) [inline recipe/meta.yaml:46]: You need to list Python in host.
```suggestion
  host:
    - python
```

File: recipe/conda_build_config.yaml
Comment (xhochy) [inline recipe/conda_build_config.yaml:1]: Remove this file and rerender

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (ilya-lavrenov) [reply recipe/meta.yaml:43]: Thanks for your suggestion! I have these requirements, but for specific output (libopenvino-python)

We have multiple outputs in this recipe, while only one of them depend on different python versions.
What is the proper way? 
- (a) Rebuild all outputs for each python versions
- (b) Rebuild only specific output which depend on the python.
I suppose if I specify python as a `host` requirement globally, CI will rebuild everything from scratch (variant (a) ). But is possible to rebuild only specific output? (variant (b) ). Is it any approach how to control which output to rebuild?

Comment (xhochy) [reply recipe/meta.yaml:43]: You should only need it on the Python specific outputs. xgboost should be a good example on how it is handled.

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Comment by ilya-lavrenov** [comment]
@conda-forge-admin, please rerender

**Pull request closed without merging**